common/api/ui-framework.api.md

```diff
@@ -4317,7 +4317,7 @@ export interface ProjectServices {
 }
 
 // @public @deprecated
-export const PromptField: import("react-redux").ConnectedComponent<typeof PromptFieldComponent, Pick<React.ClassAttributes<PromptFieldComponent> & PromptFieldProps, "ref" | "style" | "key" | "className" | "isInFooterMode" | "openWidget" | "onOpenWidget">>;
+export const PromptField: import("react-redux").ConnectedComponent<typeof PromptFieldComponent, Pick<React.ClassAttributes<PromptFieldComponent> & PromptFieldProps, "style" | "className" | "ref" | "isInFooterMode" | "key" | "onOpenWidget" | "openWidget">>;
 

```

Comment:
@calebmshafer - I have a set of these superfluous (changed order) in api files - are these a concern or just let them go?

Reply:
In directories with this problem (ui-framework and occasionally others):
npm run clean
npm run build
npm run extract-api

Reply:
Thanks Dan -- I just tried that and it seems to produce the same output... ;(

Reply:
@RBBentley what node version are you running?

Reply:
v12.16.2

Reply:
@RBBentley This happens to me sometimes even though I don't change code in the ui directories. I just revert the changes locally. If you push them I believe that you will get a CI build failure.

Reply:
Yes - I generally revert them as well... These slipped in and I guess I'll just back them out, but it sure is annoying.